src/Entity/Room.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Entity;
+
+class Room
```

Comment:
This domain object also included the building, so you should add it here aswell.